common/src/chain/tokens/mod.rs

```diff
@@ -74,36 +79,46 @@ impl OutputValue {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+pub struct TokenTransferV1 {
+    pub token_id: TokenId,
+    pub amount: Amount,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+pub struct TokenIssuanceV1 {
+    pub token_ticker: Vec<u8>,
+    pub amount_to_issue: Amount,
+    pub number_of_decimals: u8,
+    pub metadata_uri: Vec<u8>,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+pub struct TokenBurnV1 {
+    pub token_id: TokenId,
+    pub amount_to_burn: Amount,
+}
+
+//FIXME(nft_issuance): The same issue above in OutputValue.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
No. This is wrong. 

Use a Box on the large variants.

Reply:
Removed attribute